Makefile

```diff
@@ -16,7 +18,7 @@ SOURCE_AMI_ID ?= $(shell aws ec2 describe-images \
 		Name=state,Values=available \
 	--query 'max_by(Images[], &CreationDate).ImageId')
 
-AWS_DEFAULT_REGION = us-west-2
+AWS_DEFAULT_REGION ?= us-west-2
```

Comment:
@micahhausler just wondering if this variable should be renamed to `AWS_REGION`

Reply:
I think `AWS_REGION` is a better name.

Reply:
Changed to `AWS_REGION` and move it before `SOURCE_AMI_ID`.